website/src/components/landing/sections/10-tech-specs.html

```
@@ -1,108 +1,184 @@
        build emits). New specs appear here with zero edits to this file; if the
        fetch fails the section simply stays hidden. -->
-  <section class="specs" id="tech-specs" aria-label="Tech specs" hidden>
-    <style>
-      .specs{ position:relative; padding:0 0 96px; }
-      .specs-head{ padding:80px 36px 20px; display:flex; align-items:baseline; justify-content:space-between; gap:16px; flex-wrap:wrap; }
-      .specs-title{
-        font-family:'Chivo Mono', ui-monospace, monospace;
-        font-weight:500; font-size:clamp(20px, 4.2vw, 28px);
-        letter-spacing:-0.01em; line-height:1.25;
-        margin:0; text-transform:none; max-width:80%;
+<section class="specs" id="tech-specs" aria-label="Tech specs" hidden>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)